src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -216,15 +216,24 @@ public virtual TContext InitializeContextFromOptions(TOptions options, ref TCont
             context.ResultKinds = options.Kind != null ? options.ResultKinds : context.ResultKinds;
             context.FailureLevels = options.Level != null ? options.FailureLevels : context.FailureLevels;
 
-            // We initialize an aggregating logger, which includes a console logger (for general
-            // reporting of conditions that precede successfully creating an output log file).
-            context.Logger ??= InitializeLogger(context);
+            // We initialize a temporary console logger that's used strictly to emit
```

Comment:
>initialize [](http://example.com/codeflow?start=18&length=10)

The previous interim release would always set the console logger to default values, rather than honoring config or explicitly command-line configured values.